content/blog/2021-11-30-november-21-community-gems.md

```diff
@@ -0,0 +1,192 @@
+---
+title: November '21 Community Gems
+date: 2021-11-30
+description: |
+  A roundup of technical Q&A's from the DVC and CML community. This month: CML runners, working with data, DVC Studio, and more.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC and CML community. This month: CML runners, working with data, DVC Studio, and more.
+picture: 2021-11-30/nov-community-gems.png
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/november-21-community-gems/964
+tags:
+  - Data Versioning
```

Comment:
what question corresponds to this tag?

Reply:
`What's the recommended way to remove data that has been imported using `dvc import`?`

Reply:
it's very hard to connect, Data Versioning sounds too generic. If we usually tend to include DVC into gem tags, let's include `DVC` then, similar to CML? Btw, do we usually include CML as a tag - it's also too generic,. wdyt?

Reply:
These are a part of the tags we talked about listed here: https://www.notion.so/iterative/Writing-Community-Gems-3f86bbbe4b494cbb8df7dcb7c952281e#d656dfe398444e5ab9b70709a93694ae

I'll add the `DVC` tag. I think it's ok if they are a little generic as long as there is something related to the tag in the post. The purpose of tags is to show people similar types of posts and if we get too specific then we'll end up with tags that are only relevant to one post.

I'm open to other ideas though!